cli/examples/content/updateVideo.json

```diff
@@ -0,0 +1,36 @@
+{
+  "assets": [
+    {
+      "Urls": ["https://joystream.org/WorkingGroupOpening.schema.json"]
+    }
+  ],
+  "meta": {
+    "title": "Title",
+    "description": "Description",
+    "video": 1,
+    "thumbnailPhoto": "1",
+    "duration": 10,
+    "mediaPixelHeight": 20,
+    "mediaPixelWidth": 50,
+    "language": "en",
+    "hasMarketing": true,
+    "isPublic": true,
+    "isExplicit": true,
+    "personsList": [1, 2, 5],
+    "category": 2,
+    "mediaType": {
+      "codecName": "mp4",
+      "container": "zip",
+      "mimeMediaType": "mime media type"
+    },
+    "licence": {
```

Comment:
It should be licen**s**e (same thing with `createVideo.json`)

Reply:
Done https://github.com/Joystream/joystream/pull/2262/commits/ad2040b2e0dc60a082084b978cfcb83669934d14